lib/portlayer/exec/container.go

```diff
@@ -124,6 +127,9 @@ func (c *Container) Commit(ctx context.Context, sess *session.Session, h *Handle
 			res, err = tasks.WaitForResult(ctx, func(ctx context.Context) (tasks.ResultWaiter, error) {
 				return parent.CreateVM(ctx, *h.Spec.Spec(), VCHConfig.ResourcePool, nil)
 			})
+
+			// set the status to created
+			c.State = StateCreated
```

Comment:
nope - in the current incarnation the status will be updated in another function.  Ultimately, the status will most likely be removed from the portlayer and the client (persona) will derive the status from the PortLayer returned state.